pkg/ipamd/rpc_handler.go

```diff
@@ -98,37 +99,38 @@ func (s *server) DelNetwork(ctx context.Context, in *pb.DelNetworkRequest) (*pb.
 	}
 	log.Infof("Send DelNetworkReply: IPv4Addr %s, DeviceNumber: %d, err: %v", ip, deviceNumber, err)
 
-	return &pb.DelNetworkReply{Success: err == nil, IPv4Addr: ip, DeviceNumber: int32(deviceNumber)}, err
+	return &rpc.DelNetworkReply{Success: err == nil, IPv4Addr: ip, DeviceNumber: int32(deviceNumber)}, err
 }
 
 // RunRPCHandler handles request from gRPC
 func (c *IPAMContext) RunRPCHandler() error {
 	log.Info("Serving RPC Handler on ", ipamdgRPCaddress)
-
-	lis, err := net.Listen("tcp", ipamdgRPCaddress)
+	listener, err := net.Listen("tcp", ipamdgRPCaddress)
 	if err != nil {
 		log.Errorf("Failed to listen gRPC port: %v", err)
 		return errors.Wrap(err, "ipamd: failed to listen to gRPC port")
 	}
-	s := grpc.NewServer()
-	pb.RegisterCNIBackendServer(s, &server{ipamContext: c})
-	hs := health.NewServer()
-	// TODO: Implement watch once the status is check is handled correctly.
-	hs.SetServingStatus("grpc.health.v1.aws-node", healthpb.HealthCheckResponse_SERVING)
-	healthpb.RegisterHealthServer(s, hs)
+	grpcServer := grpc.NewServer()
```

Comment:
you don't like those short variable names, huh? :)

Personally, I follow the principle that the smaller the scope of the variable, the shorter the name should be. In this case, the scope of these variables is very small (single-function scope with the function only about ten lines) so I don't mind the variable names being one or two letters. For global or package-scoped variables or variables used in lengthy (20+ lines) functions and/or passed between multiple functions I usually insist on lengthy and explicit variable names

Reply:
I mostly agree, and for for-loops and similar I use short variables. I guess I'm still slightly Java-institutionalized... 😄